onadata/libs/tests/utils/test_export_builder.py

```diff
@@ -228,7 +229,9 @@ class TestExportBuilder(TestBase):
             'osm_building:building:levels': 4,
             'osm_building:ctr:lat': '23.707316084046038',
             'osm_building:ctr:lon': '90.40849938337506',
-            'osm_building:name': 'kol'
+            'osm_building:name': 'kol',
+            '_review_status': 'Active',
```

Comment:
I know it is a test, but we dont have `Active` as a possible review status.  Should we avoid any possibly misleading even in tests by using the actual expected value(s)?